### PR TITLE
Changed the color for the table headings for enough contrast with the background

### PR DESCRIPTION
**Pull request by kivuvarosekivuvan**
# Engineers Checklist
## Description
- This PR addresses the contrast issue of the table heading with the background, making it black/the color specified in the design system.
- For the inactive state, we are using `--color-light-contrast` (typically Black).
- For the hover or active state, we are using `--color-primary-50.`

## Screenshot
Screenshot 2024-07-30 at 09 15 20

## Testing
- The changes have been tested using the WAVE tool and a color contrast checker to verify the improved contrast.

### CODE REVIEW

**Comment by kivuvarosekivuvan** [comment]
@Angamanga I have changed to black but waiting for Reuben's thoughts on hover state for the `<th>`

File: apps/web-mzima-client/src/app/settings/users/users.component.scss
Comment (Ifycode) [inline apps/web-mzima-client/src/app/settings/users/users.component.scss]: You can remove the comment to clean this up @kivuvarosekivuvan 
Or is there a reason this this here?

File: apps/web-mzima-client/src/app/settings/users/users.component.scss
Comment (Ifycode) [inline apps/web-mzima-client/src/app/settings/users/users.component.scss:60]: Since this is an `scss` setting, let's use `scss` method instead of the regular `css` method of accessing children elements @kivuvarosekivuvan

**Review by Ifycode — CHANGES_REQUESTED** [review]

**Comment by kivuvarosekivuvan** [comment]
For this PR, I’m waiting for reuben’s approval of the colors. I will use
scss when he approves the colors

File: apps/web-mzima-client/src/app/settings/users/users.component.scss
Comment (kivuvarosekivuvan) [reply apps/web-mzima-client/src/app/settings/users/users.component.scss:60]: @Ifycode You mean accessing children using the ampersand operator(&)?

File: apps/web-mzima-client/src/app/settings/users/users.component.scss
Comment (kivuvarosekivuvan) [reply apps/web-mzima-client/src/app/settings/users/users.component.scss]: This was just a test, I will get rid of that

**Comment by kivuvarosekivuvan** [comment]
@Ifycode This is ready for review. Changes in already :tada:

**Review by Ifycode — APPROVED** [review]

**Pull request merged**
